### PR TITLE
Reorganize the ARCloud document structure

### PR DESCRIPTION
**Pull request by grozdanov**
- Add ARCloud Overview section
- Simplify the instructions by assembling all the steps in one page per deployment.

### CODE REVIEW

File: docs/guides/arcloud/_category_.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "label": "ARCloud",
+  "position": 11,
+  "collapsed": true,
+   "link": {
+    "type": "doc",
+    "id": "arcloud-overview"
+  }
+}
```
Comment (singlerider) [inline docs/guides/arcloud/_category_.json:9]: Please add newline at EOF

**Review by singlerider — CHANGES_REQUESTED** [review]
We do not use `.mdx` extensions. Please convert to `.md`.

File: docs/guides/arcloud/_category_.json (hunk 1 above)
Comment (grozdanov) [reply docs/guides/arcloud/_category_.json:9]: @singlerider, is there any reason not to use `.mdx` extension ?

Comment (singlerider) [reply docs/guides/arcloud/_category_.json:9]: It's not necessary to do so and I would like to keep our docs consistent with the other pages, none of which use this extension.

File: docs/guides/arcloud/_install_istio_aws.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Check the ELB address for the just created service and export it for later use:
+```sh
+export AWS_ELB_DOMAIN=$(kubectl -n istio-system get svc istio-ingressgateway --template '{{(index .status.loadBalancer.ingress 0).hostname}}')
+echo $AWS_ELB_DOMAIN
+```
+
+Modify your DNS zone by adding a CNAME entry for your domain pointing to the ELB address displayed in the previous step.
+
```
[the remark this answers is not in the document]
Comment (grozdanov) [reply docs/guides/arcloud/_install_istio_aws.mdx:8]: Where do you an issue with the formatting ?

[the remark this answers is not in the document]
Comment (singlerider) [reply docs/guides/arcloud/_install_istio_aws.mdx]: It's consistent throughout.

For example, the headings (`# Heading 1`) don't have an extra newline to give it "room to breathe."

[the remark this answers is not in the document]
Comment (singlerider) [reply docs/guides/arcloud/_install_istio_aws.mdx]: Screen Shot 2023-02-15 at 9 22 35 AM

**Review by singlerider — APPROVED** [review]

**Pull request merged**